### DISCUSSION TITLE
[Connector Request] Venly connector

### DISCUSSION THREAD

**Discussion by davidzwfu** [wevm/wagmi#3522] [Connector Request]
### What **novel use-case** does the Connector provide?

Hello,

We want to create a [Venly](https://www.venly.io/product/wallet-solutions) connector for wagmi. We have some clients that are requesting this.

Our connector will function similarly to `coinbaseWallet` as we use our own SDK: https://github.com/ArkaneNetwork/arkane-connect (FYI our company rebranded from Arkane Network to Venly but the repositories still reference Arkane).

### Are the Connector's integrations production-ready and generally available?

Yes they have been generally available for already >4 years: https://github.com/ArkaneNetwork/arkane-connect

They are also available on different CDN providers (ex: [cdnjs](https://cdnjs.com/libraries/venly-connect))

### Are you committed to actively mainting the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by davidzwfu** [comment]
@tmm @jxom Can we please get an update on this? Thanks

**Reply by jxom** [reply]
Hi!

We are starting to only accept connector PR requests from sponsors – this covers initial setup review, due diligence, and ongoing maintenance from the core team so we can keep the quality bar very high. More info here on sponsoring: https://github.com/sponsors/wevm. Happy to answer any questions here or set up time to chat further.

Additionally, it would be great if we could remove the attack vector of remote URI popups: https://github.com/ArkaneNetwork/arkane-connect/blob/f6fdca34703ab042522867fd12bbfc4ed44ee243/src/popup/GeneralPopup.ts#L11

**Reply by davidzwfu** [reply]
Hi jxom,

Could you please elaborate on the attack vector? As the popup is only used to open/communicate with our web app. The URI is not based on user input of any sort.

**Reply by jxom** [reply]
Got it.
